llvm/test/CodeGen/DirectX/legalize-lifetimes.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: opt -S -passes='dxil-op-lower' -mtriple=dxil-pc-shadermodel6.3-library %s | FileCheck %s
+
+; CHECK-NOT: error: Unsupported intrinsic llvm.lifetime.start.p0 for DXIL lowering
+; CHECK-NOT: error: Unsupported intrinsic llvm.lifetime.end.p0 for DXIL lowering
```

Comment:
Do we really need to check-not an error? If it errors `opt` will return non-zero and fail the test right?

Reply:
You are right. I wanted to distinguish from the current behavior but thats not needed.